github4s/src/test/scala/github4s/utils/TestData.scala

```diff
@@ -49,19 +49,20 @@ trait TestData extends DummyGithubUrls {
   val validCode           = "code"
   val invalidCode         = "invalid-code"
 
-  val validRepoOwner     = "47deg"
-  val invalidRepoOwner   = "invalid47deg"
-  val validRepoName      = "github4s"
-  val invalidRepoName    = "GHInvalidRepoName"
-  val validRedirectUri   = "http://localhost:9000/_oauth-callback"
-  val validPage          = 1
-  val invalidPage        = 999
-  val validPerPage       = 100
-  val validFilePath      = "README.md"
-  val invalidFilePath    = "NON_EXISTENT_FILE_IN_REPOSITORY"
-  val validDirPath       = "lib"
-  val validSymlinkPath   = "bin/some-symlink"
-  val validSubmodulePath = "test/qunit"
+  val validRepoOwner             = "47deg"
+  val invalidRepoOwner           = "invalid47deg"
+  val validRepoName              = "github4s"
+  val validRepoNameWithMilestone = "marlow"
```

Comment:
not sure we want to use a private repo

Reply:
This info is private @anamariamv

Reply:
Sorry, I used marlow because it returns a result with data and I forgot to change it, thanks.